backend/app/adapter/google/account_integration_test.go

```diff
@@ -0,0 +1,95 @@
+// +build !integration all
```

Comment:
```suggestion
// +build integration all
```
You should mark it as an integration test instead of unit test.

Reply:
if that's the case, github/account_integration_test.go line 1 should be changed as well, right

Reply:
@dawntomm Good catch! Will open a PR to address that soon.

Reply:
https://github.com/short-d/short/pull/481